.github/workflows/typescript.yml

```
@@ -1,25 +1,64 @@
+      - name: Run linting
+        run: bun run lint
+      - name: Check formatting
+        run: bun run prettier
+
+  type-check:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - uses: oven-sh/setup-bun@22457c87c1b161cf7dde222c3e82b2b5f8d2bed2 # v2
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)